pkgs/by-name/ko/komac/package.nix

```
@@ -7,32 +7,42 @@
 , darwin
 , testers
 , komac
+, dbus
+, zstd
 }:
 
 let
-  version = "2.2.1";
+  version = "2.4.1";
   src = fetchFromGitHub {
     owner = "russellbanks";
     repo = "Komac";
     rev = "v${version}";
-    hash = "sha256-dPX8/JUQ+vugd+M/jIjBf4/sNbac0FVQ0obhyAAGI84=";
+    hash = "sha256-AltHTa4Jvd9zG/BVAhyK2I3Bqbbt2bbNKsfmI4mNqa4=";
   };
 in
 rustPlatform.buildRustPackage {
   inherit version src;
 
   pname = "komac";
 
-  cargoHash = "sha256-CDPN90X3m/9FRLolAVCIcAuajZbB5OAgLcFXq2ICS8g=";
+  cargoHash = "sha256-Csn/vdWgOHxHVhUPT2FHq2/o/FN2hyZxtRRxYfN+42U=";
 
-  nativeBuildInputs = lib.optionals stdenv.isLinux [ pkg-config ];
+  nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = lib.optionals stdenv.isLinux [
+  buildInputs = [
+    dbus
     openssl
+    zstd
   ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
     darwin.apple_sdk.frameworks.SystemConfiguration
   ];
 
+  env = {
+    OPENSSL_NO_VENDOR = true;
+    ZSTD_SYS_USE_PKG_CONFIG = true;
```

I can reproduce the yara-x checksum mismatch errors as same as your uploaded screenshot. 
So how about adding this environment variable? I can build in my NixOS after this change.

```suggestion
    ZSTD_SYS_USE_PKG_CONFIG = true;
    YRX_REGENERATE_MODULES_RS = "no";
```

https://github.com/VirusTotal/yara-x/blob/c34873e807a95ea2fc2154d6711044dadbcd76d8/.github/workflows/release.yaml#L205-L210

---

Nice! This fixes the error. Tysm for finding out this solution